app/validators/amazon_validator.rb

```diff
@@ -0,0 +1,9 @@
+class AmazonValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
```

Comment:
Unused method argument - `record`. If it's necessary, use `_` or `_record` as an argument name to indicate that it won't be used.<br>Unused method argument - `attribute`. If it's necessary, use `_` or `_attribute` as an argument name to indicate that it won't be used.